routes/api/account.js

```diff
@@ -103,4 +103,63 @@ router.post('/details', async (req, res) => {
             title: undefined //Deprecated
         }
     });
+});
+
+router.post('/details-multiple', async (req, res) => {
+    const usersArray = req.body.users;
+    if (typeof usersArray != 'object') {
+        res.json({
+            error: 1,
+            errorMessage: 'Invalid users array'
+        });
+        return;        
+    }
+
+    if (usersArray.length < 1 ) {
+        res.json({
+            error: 1,
+            errorMessage: 'Invalid users array'
+        });
+        return;
+    }
+
+
+    let userReturn = [];
+    for (let i = 0; i < usersArray.length; ++i) {
+        let userId = parseInt(usersArray[i].userId, 10);
+        
+        if (isNaN(userId)) {
+            console.log('userID NaN');
+            continue;
+        };
+
+        let user;
+        
+        try {
+            user = await Users.get(userId);
+        } catch (error) {
+            log.error(error);
+            continue;
+        }
+
+        if (!user) {
+            continue;
+        }
+
+        let singleUser = [{
+            users: user.getId(),
+            name: user.getName(),
+            emailAddress: undefined, //Deprecated
+            joinDate: user.getCreated(),
+            joinTimestamp: user.getCreated() ? Math.round(user.getCreated().getTime() / 1000) : undefined,
+            coinsBalance: user.getCoins().getBalance(),
+            profile: {
+                photo: user.getAvatar(),
+                photoThumb: user.getAvatarThumb(),
+                title: undefined //Deprecated
+            }
+        }];
+        userReturn.push(singleUser);
+    }
+    res.json({userReturn});
```

Comment:
```suggestion
    res.json({users: userReturn});
```
Better API output syntax